public/data/ibrahimrehman1.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Ibrahim Rehman",
+  "type": "personal",
+  "bio": "Full Stack Web and Mobile App Developer",
+  "avatar": "https://github.com/ibrahimrehman1.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/ibrahimrehman1",
+      "icon": "github"
+    },
+    {
+      "name": "Connect with me on Linkedin",
+      "url": "https://www.linkedin.com/in/ibrahimrehmanweb/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Ibrahim02572102",
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Remote Work",
+      "date": "April 2022",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Started working at ConceptZi in part-time, as a Full Stack Web and Mobile App Developer. Mostly working on React, Vue and Flutter",
+      "url": "https://www.eddiejaoude.io/"
```

Comment:
Are you sure, you wanted to add Eddie's URL here?
I think you should update it with your URL or you can safely remove line no. `30`, as it's not compulsory to add a URL to milestones! 



__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inelegant

Reply:
Can you please check now? I have made the changes.